docs/source/joblib.rst

```diff
@@ -52,6 +52,15 @@ validated parameter search as follows.
    with parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
        search.fit(digits.data, digits.target)
 
+## Nested Parallelism
+
+Note that this code will not work if you distribute it via dask in order to implement nested parallelism. The reason is that each worker will have a fresh copy of sklearn and `from sklearn.externals.joblib import parallel_backend` will not apply to them, resulting in a "KeyError" when trying to distribute, but no problem when running the inner loop locally. The solution is 
+
+.. code-block:: python
+
+    from distributed.joblib import DistributedBackend 
+    register_parallel_backend('distributed', DistributedBackend, make_default=True)
```

Comment:
Right, but the situation I had was a 16-core machine where I was running 8 independent runs of 2-fold crossvalidation in parallel, and as far as I can tell since 8*2 = 16 there should be no waiting

In any case,  the problem that made me switch to the second version was not that tasks were waiting, it was that the first version was unstable (i.e. continually caused pandas to crash)